static/js/program_course_list.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+/* global SETTINGS:false */
+import ProgramCourseList from './components/ProgramCourseList';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+import getMuiTheme from 'material-ui/styles/getMuiTheme';
+
+const courseListEl = document.querySelector('#program-course-list');
+
+const ThemedProgramCourseList = () => (
+  <MuiThemeProvider muiTheme={getMuiTheme()}>
+    <ProgramCourseList courses={SETTINGS.courses} />
+  </MuiThemeProvider>
```

Comment:
the JSX here could be inlined in the `ReactDOM.render` call below, although it doesn't make a big difference